components/text-utilities/text.css

```diff
@@ -0,0 +1,80 @@
+@import url('/components/css/common.css');
+.text-display{
+    display: flex;
+    flex-direction: column;
+    border-radius: 0.2rem;
+    background-color: var(--white);
+    margin:1rem;
+    padding: 1rem;
+    width:70%;
+}
+.h-1{
+    font-size: 3rem;
+    line-height: 4rem;
+    
+}
+.h-2{
+    font-size: 2rem;
+    line-height: 2.5rem;
+}
+.h-3{
+    font-size: 1.5rem;
+    line-height: 2rem;
+}
+.h-4{
+    font-size: 1rem;
+    line-height: 1.5rem;
+}
+.h-5{
+    font-size: 0.8rem;
+    line-height: 1rem;
+}
+.text-xl{
+    font-size: 3rem;
+    line-height: 4rem;
+    margin: 0px;
+}
+.text-lg{
+    font-size: 2rem;
+    line-height: 2.5rem;
+    margin: 0px;
+}
+.text-md{
+    font-size: 1.5rem;
+    line-height: 2rem;
+    margin: 0px;
+}
+.text-sm{
+    font-size: 1rem;
+    line-height: 1.5rem;
+    margin: 0px;
+}
+.text-xs{
+    font-size: 0.8rem;
+    line-height: 1rem;
+    margin: 0px;
+}
+.text-left{
+    text-align: left;
+}
+.text-center{
+    text-align: center;
+}
+.text-right{
+    text-align: right;
+}
+.light{
+    font-weight: 200;
+}
+.regular{
+    font-weight: 400;
+}
+.semibold{
+    font-weight: 600;
+}
+.bold{
+    font-weight:700;
+}
+.black{
+    font-weight:900;
```

Comment:
Naming could be better for font weights like `.fw-light .fw-regular .fw-semibold .fw-bold .fw-bolder`. Just a suggestion. Everything else looks good to me

Reply:
Thank you siddhi for the suggestion it is more readable now ,I  have made the changes and will merge the PR